src/game/GameScreen.js

```diff
@@ -120,3 +118,29 @@ export default class GameScreen extends React.Component {
         }
     };
 }
+const EnhancedGameScreen = withState('loading', 'setLoading', true)(GameScreen);
```

Comment:
Why using recompose for that? Can't it be included in the `GameScreen` directly ?

Reply:
Actually, I saw that with julien
At begining, recompose was here to manage an upper state to display dynamic loader inside the navigationOptions tab bar. (Wich loads before the state is created and never update after state creation)
But it doesn't work better because recompose doesn't reinject loading changes in navigationOptions props.
Finally, I inject manually the loading in this.props when it has to change, and navigationOptions is created with this.props.
I just keep recompose because it's a cool way to export components with addictionnals properties when needed
But I can be removed if you find this useless.

Reply:
I think it is overkill here. Moreover, HOC can leads to many other issues so I try to avoid them when possible

Reply:
OK
I hope @jdemangeon wont be too disappointed